pyvalidator/is_mailto.py

```diff
@@ -0,0 +1,18 @@
+import re
+
+def is_valid_mailto_link(link):
```

Comment:
Name the function the same as the name of the file.

This is kinda a convention that the project is following.